build.gradle

```diff
@@ -6,19 +6,26 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:4.1.2"
+        implementation 'com.github.euphony-io:euphony:562a8fab8d'
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
-
 allprojects {
     repositories {
         google()
         jcenter()
+        maven { url 'https://jitpack.io' }
     }
 }
-
 task clean(type: Delete) {
     delete rootProject.buildDir
+}
+android {
+    compileSdkVersion 19
+    buildToolsVersion '19.1.0'
+}
+dependencies {
+    compile project(':app')
```

Comment:
There seems to be a build error in this part as well.
I don't know Android well yet, so I don't know why you wrote this.
If you don't need this part after editing it like the above, you can delete it.

Reply:
Thanks for review! I fixed all you said!
And by mistake, I pressed "rebase and merge" button so hope this make any problem.